content/blog/management-paradigms-for-virtual-machines-running-on-kubernetes.md

```diff
@@ -0,0 +1,374 @@
+my-vm-0   13s   Provisioning   False
+my-vm-0   29s   Starting       False
+my-vm-0   42s   Running        False
+my-vm-0   42s   Running        True
+```
+
+Once the VM is running, it’s possible to login with the SSH identity and hostname given to the VM (assuming DHCP registers the hostname in DNS on the management network).
+
+```shell
+$ ssh fedora@my-vm-0
+[fedora@my-vm-0 ~]$
+```
+
+So, what does the VM instance actually look like? Let’s install some tools and inspect.
+
+```shell
+$ sudo dnf install -yq fastfetch virt-what
+$ sudo virt-what
+redhat
+kvm
+$ fastfetch --pipe --localip-show-ipv4 false
+             .',;::::;,'.                 fedora@my-vm-0
+         .';:cccccccccccc:;,.             --------------
+      .;cccccccccccccccccccccc;.          OS: Fedora Linux 41 (Cloud Edition) x86_64
+    .:cccccccccccccccccccccccccc:.        Host: KubeVirt (RHEL-9.4.0 PC (Q35 + ICH9, 2009))
+  .;ccccccccccccc;.:dddl:.;ccccccc;.      Kernel: Linux 6.11.4-301.fc41.x86_64
+ .:ccccccccccccc;OWMKOOXMWd;ccccccc:.     Uptime: 7 mins
+.:ccccccccccccc;KMMc;cc;xMMc;ccccccc:.    Packages: 550 (rpm)
+,cccccccccccccc;MMM.;cc;;WW:;cccccccc,    Shell: bash 5.2.32
+:cccccccccccccc;MMM.;cccccccccccccccc:    Terminal: /dev/pts/0
+:ccccccc;oxOOOo;MMM000k.;cccccccccccc:    CPU: Intel Core (Haswell, no TSX, IBRS) @ 2.60 GHz
+cccccc;0MMKxdd:;MMMkddc.;cccccccccccc;    GPU: Unknown Device 1111 (VGA compatible)
+ccccc;XMO';cccc;MMM.;cccccccccccccccc'    Memory: 435.27 MiB / 3.80 GiB (11%)
+ccccc;MMo;ccccc;MMW.;ccccccccccccccc;     Swap: 0 B / 3.80 GiB (0%)
+ccccc;0MNc.ccc.xMMd;ccccccccccccccc;      Disk (/): 805.66 MiB / 62.92 GiB (1%) - btrfs
+cccccc;dNMWXXXWM0:;cccccccccccccc:,       Locale: en_US.UTF-8
+cccccccc;.:odl:.;cccccccccccccc:,.
+ccccccccccccccccccccccccccccc:'.
+:ccccccccccccccccccccccc:;,..
+ ':cccccccccccccccc::;,.
+```
+
+Except for the “Host” hint, this looks like any VM instance on a KVM hypervisor.
+
+With `virtctl` it’s possible to live migrate, pause/unpause, stop/start and restart the VM. Deleting the VM requires `kubectl`.
+
+```shell
+kubectl delete -n hpe-vmi vm/my-vm-0
+```
+
+This will remove all resources created with `virtctl`, including `PVCs`.
+
+# UX with Web UIs
+
+KubeVirt does not have an official graphical user interface. That is a tall threshold for new users who are familiar with legacy VM management solutions where everything is a right-click away, structured in an intuitive manner. In a way, the KubeVirt project assumes the user to have fundamental KVM knowledge and able to scrape by managing Kubernetes resources through the CLI.
+
+Fortunately, there are KubeVirt implementations that heavily focus on a graphical user experience and provide a great way to learn and explore the capabilities, very similar to legacy hypervisors.
+
+We’ll take a closer look at OKD, the upstream Kubernetes distribution of OpenShift, and Harvester, an HCI solution built for VMs on KubeVirt with striking simplicity.
```

Comment:
Suggestion to spell out HCI the first time you mention it in the post:

We’ll take a closer look at OKD, the upstream Kubernetes distribution of OpenShift, and Harvester, an Hyper Converged Infrastructure (HCI) solution built for VMs on KubeVirt with striking simplicity.

You can then use the abbreviation HCI in the sentence:

Harvester is an open source HCI solution primarily focused on running a highly opinionated stack of software and tools on Kubernetes designed solely for running VMs. Harvester can be consumed by Rancher to allow Rancher to deploy and manage Kubernetes clusters on Harvester in a symbiotic relationship.